Source/ACE.Entity/ObjectGuid.cs

```diff
@@ -5,14 +5,71 @@ namespace ACE.Entity
     public enum GuidType
     {
         Undef,
+        Weenie,
+        Static,
+        Generator,
         Player,
-        Creature
+        Creature,
+        Item
     }
 
     public struct ObjectGuid
     {
         public static readonly ObjectGuid Invalid = new ObjectGuid(0);
 
+        public static uint WeenieMin { get; } = 0x00000001;
+        public static uint WeenieMax { get; } = 0x000F423F;
+
```

Comment:
Ahh, I didn't realize 0xF423F was 999,999.  Would it be possible to choose our ranges based on hex values (e.g. 0x100000) instead of decimal -- mixing decimal and hex here obfuscates the meaning of the code (e.g. I thought it was a tight range, when in reality it was an arbitrary decimal value converted to hex).  If not, can you comment it, instead of having some large, arbitrary magic number there?